packages/@expo/cli/src/start/startAsync.ts

```diff
@@ -76,9 +76,23 @@ export async function startAsync(
   const platformBundlers = getPlatformBundlers(exp);
 
   if (!options.forceManifestType) {
-    const easUpdatesUrlRegex = /^https:\/\/(staging-)?u\.expo\.dev/;
-    const isEasUpdatesUrl = exp.updates?.url ? easUpdatesUrlRegex.test(exp.updates.url) : false;
-    options.forceManifestType = isEasUpdatesUrl ? 'expo-updates' : 'classic';
+    if (exp.updates?.useClassicUpdates) {
+      options.forceManifestType = 'classic';
+    } else {
+      const classicUpdatesUrlRegex = /^(staging-)?exp\.host/;
```

Comment:
exp.host uses a "." for staging:
```suggestion
      const classicUpdatesUrlRegex = /^(staging\.)?exp\.host$/;
```